### ISSUE TITLE
Couldn’t work out with the DevTool

### ISSUE THREAD

**Issue by backpack-operations** [Laravel-Backpack/community-forum#921]
Hello team,

I just received this on email:
"Hi,

I really liked Backpack but couldn’t work out with the DevTools. It seems so buggy on model creations especially when I try to add fields with foreign ids (or belongs to, it doesn’t matter)

Admin panel works perfectly. I use Herd to handle php and other stuff for me. I use MySQL. Recently ran the composer update command.

Here is the version command output.

### PHP VERSION:
8.3.3

### PHP EXTENSIONS:
Core, date, libxml, openssl, pcre, sqlite3, zlib, bcmath, bz2, calendar, ctype, curl, dba, dom, hash, FFI, fileinfo, filter, ftp, gd, gmp, json, iconv, SPL, session, intl, standard, ldap, mbstring, mysqlnd, mysqli, pcntl, exif, imap, PDO, pdo_mysql, pdo_pgsql, pdo_sqlite, pgsql, Phar, posix, random, readline, redis, Reflection, imagick, shmop, SimpleXML, soap, sockets, sodium, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlwriter, xsl, zip, Zend OPcache

### LARAVEL VERSION:
11.2.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.3.0
backpack/crud: 6.7.5
backpack/devtools: 3.1.1
backpack/generators: v4.0.5
backpack/pro: 2.1.12
backpack/theme-tabler: 1.2.8

Can you help me about it? Am I doing something wrong with the installation? I just used the composer to install everything on a fresh project.

Thanks!

"

**Comment by jcastroa87** [comment]
Hello,

Could you please provide more details about the issue you're facing? It would be helpful if you could share screenshots or additional information to better understand the problem you're encountering. Thank you for your cooperation.

Cheers.

**Comment by jcastroa87** [comment]
Hello, do you have any updates on the issues with Devtools? We'd like to investigate further. Thank you.

**Comment by luckylads** [comment]
Hi,

I am running in a similar situation. I have bought backpack pro & dev-tools but that's not working as intended.

For example, Model creation with a belongsto throws a crash
![image](https://github.com/Laravel-Backpack/community-forum/assets/94736602/f480ba9b-5b70-4d67-8969-2c81692eb4cf)
![image](https://github.com/Laravel-Backpack/community-forum/assets/94736602/527fd4fb-6c55-40b0-8d5a-7c0f8742d8ed)


Moreover, I downgraded to backpack 3.0.1 where this works, but the fields are not showing correctly. See screenshot with a belongsto. The fields stay as a textbox
![image](https://github.com/Laravel-Backpack/community-forum/assets/94736602/f994248f-94c6-4f5d-8761-96a46e05ab7d)

**Comment by backpack-operations** [comment]
https://www.youtube.com/watch?v=kyIFrTS1eb0 @jcastroa87, Hi, the customer just sent me this video in relation with the bug!

**Comment by jcastroa87** [comment]
Thanks @luckylads i confirm this bug:

<img width="1464" alt="Screenshot 2024-04-22 at 3 34 28 PM" src="https://github.com/Laravel-Backpack/community-forum/assets/44643608/2209ecf8-c5ca-4924-8a63-75be7a0dd18d">

<img width="1475" alt="Screenshot 2024-04-22 at 3 34 11 PM" src="https://github.com/Laravel-Backpack/community-forum/assets/44643608/48871b06-82ed-422d-b42b-be8b658d4832">

@backpack-operations i will check about video now and let you know.

Meanwhile i will ask to @pxpm to check about the first bug.

Cheers.

**Comment by jcastroa87** [comment]
I confirm the second bug with "ForeignId", when this is selected bring to front this error page:

<img width="1461" alt="Screenshot 2024-04-22 at 3 37 46 PM" src="https://github.com/Laravel-Backpack/community-forum/assets/44643608/026ceaeb-97bd-409d-82be-27739ab7a8ef">

@pxpm it was test on this versions:
```
### PHP VERSION:
8.3.6

### PHP EXTENSIONS:
Core, date, libxml, openssl, pcre, sqlite3, zlib, bcmath, bz2, calendar, ctype, curl, dba, dom, hash, FFI, fileinfo, filter, ftp, gd, gmp, json, iconv, SPL, session, intl, standard, ldap, mbstring, mysqlnd, mysqli, pcntl, exif, imap, PDO, pdo_mysql, pdo_pgsql, pdo_sqlite, pgsql, Phar, posix, random, readline, redis, Reflection, imagick, shmop, SimpleXML, soap, sockets, sodium, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlwriter, xsl, zip, Zend OPcache

### LARAVEL VERSION:
11.4.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.3.3
backpack/crud: 6.7.10
backpack/devtools: 3.1.1
backpack/generators: v4.0.5
backpack/pro: 2.1.13
backpack/theme-tabler: 1.2.8
```

Cheers.

**Comment by pxpm** [comment]
Thanks @jcastroa87 and @luckylads for the bug confirmation.

I've released devtools 3.1.3 that should fix this. 

Let me know if you guys still experiencing issues. 

Cheers
